### PR TITLE
Update space-age test suite

### PR DESCRIPTION
**Pull request by BNAndras**
There are eight community solutions for this exercise, and seven of them would break if we add the invalid planet test because they only check the eight atoms. That also includes the CI example so I think it's easier to mark the invalid planet test as not included.

### CODE REVIEW

**Comment by kotp** [comment]
And yet there are only 8 solutions.  Should we?  But then on the other side of the problem, if you want to explore adding another celestial body that orbits, should we stop that from happening?

File: exercises/practice/space-age/.meta/tests.toml [hunk 1]
```diff
@@ -32,3 +32,7 @@ description = "age on Uranus"
 
 [80096d30-a0d4-4449-903e-a381178355d8]
 description = "age on Neptune"
+
+[57b96e2a-1178-40b7-b34d-f3c9c34e4bf4]
+description = "invalid planet causes error"
+include = false
```
Comment (kotp) [inline exercises/practice/space-age/.meta/tests.toml:38]:
```suggestion
include = false

```

Missing EOL for that last line.

**Review by kotp — APPROVED** [review]
Approving, but consider.  I like it in general without a check for whatever.  Unless we enforce that without question only planets have the idea of rotating around the sun to complete a year, it does not make sense to invalidate something that behaves like other things in this list that orbit the sun.

**Comment by BNAndras** [comment]
The CI breaks on occasion like this. For a while it was binary, but this time it's phone-number. Erik and I couldn't see anything obviously wrong with the exercises so I think it's some part of the make process that hiccups.

But yeah, I have similar reservations about the sun test case. For starters, the sun isn't a planet but a star so saying "invalid planet" is misleading. Is it invalid because it's not a planet? Is it because it's not on the exercise's list of valid planets? Or is a valid planet just one that orbits the sun like Earth does? Then sun is excluded because it orbits the center of the Milky Way Galaxy and not itself.

Honestly, the canonical test could have used a clearly fictional planet like "Planet X", but I'm not sure it's worth the churn across all the tracks at this point.

**Review by kotp — APPROVED** [review]

**Pull request merged**